### ISSUE TITLE
Download EPUB files in the background

### ISSUE THREAD

**Issue by pmartin** [pmartin/plop-reader#57]
For now, EPUB version of entries are downloaded in the foreground, while syncing -- and it works rather fine.

But downloading those files takes time ; and, as the sync is blocking, the user cannot read anything while we are downloading EPUB version of the most recent entries (the ones we are syncing).
It's quite frustrating when no sync has been done for some time (or on first sync ever), as _many_ entries are fetched: downloading EPUB can then take up to a few minutes!

A better way would be to download EPUB version of entries in the background, once the other steps of sync are done. This way, the user could read entries for which EPUB files have already been downloaded before, or the HTML version of entries for which the EPUB version has not been downloaded yet.

**Comment by pmartin** [comment]
Downloading in the **background** is quite complex... And I think it will add too much complexity to the application, for a gain that's not that important.
=> At least for now, I won't do downloads in the background.

Instead, I'll do downloads in the foreground (when syncing), but with several downloads in parallel, to speed things up.
